### DISCUSSION TITLE
How to use the `not` parameter in an API call?

### DISCUSSION THREAD

**Discussion by manuelgrabowski** [python-gitlab/python-gitlab#2473] [Q&A] [answered]
Feeling a bit silly… 😅 

The https://docs.gitlab.com/ee/api/issues.html#list-issues endpoint has a "negating" parameter called `not`:

> Return issues that do not match the parameters supplied.

I'm trying to use that, but `not` is a reserved word in Python and I wasn't able to find a way around this yet. Can I supply the parameters to the `list()` call in some way that makes it possible to specify this parameter?

**Comment by nejch** [comment, marked as answer]
Hey @manuelgrabowski!

When API attributes conflict with python or python-gitlab, you can use the `query_parameters` dict instead. See the docs here:
https://python-gitlab.readthedocs.io/en/stable/api-usage.html?highlight=query_parameters#managers

**Reply by manuelgrabowski** [reply]
Thanks a lot, works like a charm – really appreciate your super quick and kind response to what probably was more of a Python beginner than a `python-gitlab` question.
